node/pkg/driver/node_utils.go

```diff
@@ -239,6 +243,25 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
+func (n NodeUtils) IsFCExists() bool {
+	if n.IsPathExists(FCPath) {
+		return n.IsDirHasContent(FCPath)
+	}
+	return false
```

Comment:
We can simplify this by:
```suggestion
	return n.IsPathExists(FCPath) && return n.IsDirHasContent(FCPath)
```